userspace/libsinsp/event.cpp

```diff
@@ -764,7 +764,7 @@ Json::Value sinsp_evt::get_param_as_json(uint32_t id, OUT const char** resolved_
 	//
 	// Get the parameter
 	//
-	sinsp_evt_param *param = &(m_params[id]);
+	sinsp_evt_param *param = &(m_params.at(id));
 	payload = param->m_val;
 	payload_len = param->m_len;
```

Comment:
I think we might want to drop the
`ASSERT(id < get_num_params());` line above now!